Documentation/ApiOverview/Context/_MyControllerServices.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+  MyVendor\MyExtension\:
+    resource: '../Classes/*'
+    exclude: '../Classes/Api/*'
```

Comment:
What is Classes/Api?

Additionally, apart from "Api" this is the default Services.yaml which every extension should have. So no need to mention it here.

Reply:
This is just an example. Api can be anything you want.
You know what you need. Other do not know it!

Reply:
People tend to copy/paste code with less thinking. Additionally, this overrides the Classes/Domain/Model exclusion which is recommended.

Reply:
Does this mean I shall add more folders to the exclude part?

Reply:
This is the default which is recommended at least:
https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/DependencyInjection/Index.html#dependency-injection-in-extensions
Models should be excluded as they must not be injected as they are prototypes, not services.

Reply:
I don't get why an exclusion is relevant for injecting the Context at all?

Reply:
This is needed because DI tends to throw exceptions. See the discussion on Slack.

https://typo3.slack.com/archives/C025BQLFA/p1709064210378719?thread_ts=1709043688.435639&cid=C025BQLFA

Reply:
This has cost me many hours until I found the possibility to exclude a path.

Reply:
So you exclude some random classes while the PageArguments cannot be injected. I read that and I don't understand the reason behind this. IMHO this does not really solve the problem but hides the problem.

Reply:
I have renamed it from Api into FolderToExclude.
This is just a simple example for the user to get a short and fast introduction without any sophisticated lecture. 
So no more deeper explanations belong to this page. Please understand this! 
And besides I am not able to discuss with you the very complicated issue with the page parameter Exception here. Does it belong here?